addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -828,13 +828,16 @@ body.editor_enable.editor_has_snippets {
                 }
             }
             .o_we_pager_next, .o_we_pager_prev {
+                margin: 3px;
                 padding: $o-we-sidebar-content-field-label-spacing;
                 cursor: pointer;
+                border: 1px solid;
```

Comment:
I prefer when the color argument is not omitted, write `currentColor` explicitely